### PR TITLE
Make transform explicit in XML files of IMUInverseKinematicsTool 

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue #0
### Brief summary of changes
Made builtin transform in IMUInvereseKinematicsTool identity and changed property comment and setup file for test accordingly
### Testing I've completed
Ran tests
### CHANGELOG.md (choose one)

- no need to update because internal

The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/?C=N;O=D; click the folder whose name is this PR's number.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2626)

### CODE REVIEW

**Comment by aymanhab** [comment]
@chrisdembia this could be a quick one. Thank you

File: OpenSim/Simulation/OpenSense/IMUInverseKinematicsTool.h [hunk 1]
```diff
@@ -75,7 +75,7 @@ OpenSim_DECLARE_CONCRETE_OBJECT(IMUInverseKinematicsTool, Object);
 
     OpenSim_DECLARE_PROPERTY(sensor_to_opensim_rotations, SimTK::Vec3,
             "Space fixed Euler angles (XYZ order) from IMU Space to OpenSim."
-            " Default fom IMU world (Z up) to OpenSim (Y up, X forward)");
+            " Default to Identity");
```
Comment (chrisdembia) [inline OpenSim/Simulation/OpenSense/IMUInverseKinematicsTool.h]: I think we could improve the clarity by changing this to "Default: (0, 0, 0)."

File: Applications/opensense/test/setup_IMUInverseKinematics_HJC_trial.xml [hunk 2]
```diff
@@ -3,6 +3,8 @@
 	<IMUInverseKinematicsTool name="track_HJC_trial">
 		<!--The accuracy of the solution in absolute terms, i.e. the number of significant digits to which the solution can be trusted. Default 1e-6.-->
 		<accuracy>9.9999999999999995e-07</accuracy>
+		<!-- Space fixed Euler angles (XYZ order) from IMU Space to OpenSim. Default to Identity. -->
```
Comment (tkuchida) [inline Applications/opensense/test/setup_IMUInverseKinematics_HJC_trial.xml]: FYI, comment here doesn't match new property definition line in IMUInverseKinematicsTool.h.

Comment (aymanhab) [reply Applications/opensense/test/setup_IMUInverseKinematics_HJC_trial.xml]: Thanks @tkuchida Indeed I didn't recreate the example file but I went ahead and fixed it by hand. Nice catch

**Comment by aymanhab** [comment]
@jimmyDunne Once this is merged we need to update the matlab scripts and public example for opensense. Please let me know if you need help with that.

**Comment by aymanhab** [comment]
Thanks @chrisdembia and @tkuchida for the prompt review 👍

**Pull request merged**